src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/xlf/MicrosoftCodeQualityAnalyzersResources.cs.xlf

```diff
@@ -718,7 +718,7 @@
         <note />
       </trans-unit>
       <trans-unit id="OverloadOperatorEqualsOnOverridingValueTypeEqualsMessage">
-        <source>Implement the equality operators and make their behavior identical to that of 'Equals' method</source>
+        <source>Implement the equality operators and make their behavior identical to that of the Equals method</source>
```

Comment:
I think xlf files shouldn't be manually modified to allow the localized messages to be reviewed again after this change.

Reply:
Thanks! Done.